projects/storefrontstyles/scss/layout/page-templates/_order-detail-page.scss

```diff
@@ -18,6 +18,17 @@
       margin-inline-start: 0;
       margin-inline-end: 0;
 
+      // TFM style changes
+      @include forVersion(2.1) {
```

Comment:
Since we deliver MyOrg in major version, I hope we don't need TFM here (or at least change it to 3.0)?
@tobi-or-not-tobi @developpeurweb

Reply:
yes, but it will not harm. I suggest we do this in a follow up activity.

Reply:
Created https://github.com/SAP/spartacus/issues/9044 for non my-company related usage